data-donation-sdk/src/commonMain/kotlin/care/data4life/datadonation/donation/program/model/ProgramModel.kt

```diff
@@ -31,52 +31,60 @@ enum class BlurFunction(val value: String) {
 }
 
 @Serializable(with = BlurFunctionSerializer::class)
-enum class RevocationMode(val value: String) {
+internal enum class RevocationMode(val value: String) {
     DELETE("delete"),
     ANONYMIZE("anonymize")
 }
 
 @Serializable
-internal data class ProgramResourceBlurItem(
+internal data class QuestionnaireResponseItemBlur(
     val linkId: String,
     @SerialName("fn")
     @Contextual
     val function: BlurFunction
 )
 
 @Serializable
-internal data class ProgramResourceBlur(
-    val location: String? = null,
+internal data class ProgramFhirResourceBlur(
+    @SerialName("location")
+    val targetTimeZone: String? = null,
     @Contextual
-    val authored: BlurFunction? = null,
-    val items: List<ProgramResourceBlurItem>
+    @SerialName("authored")
+    val questionnaireResponseAuthored: BlurFunction? = null,
+    @SerialName("items")
+    val itemBlurs: List<QuestionnaireResponseItemBlur>
 )
 
 @Serializable
-internal data class ProgramResource(
+internal data class ProgramFhirResourceConfiguration(
     val url: String,
     val versions: List<String>? = null,
-    val blur: ProgramResourceBlur? = null
+    @SerialName("blur")
+    val fhirBlur: ProgramFhirResourceBlur? = null
 )
 
 @Serializable
-internal data class ProgramAnonymizationBlur(
-    val location: String,
+internal data class ProgramAnonymizationGlobalBlur(
```

Comment:
Do we actually need the word Anonymization in those models? i have the feeling `ProgramGlobalBlurRule` would work already?

Also if this is just for questionnaireResponse i would mention that. maybe 
`GlobalQuestionnaireResponseBlurRule` (in opposition to the `QuestionnaireResponseBlurRule`)

Reply:
Changed it